sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_deserialize.py

```diff
@@ -101,6 +107,9 @@ def from_blob_properties(blob_properties, **additional_args):
 
     # Parse additional Datalake-only properties
     file_props.encryption_context = additional_args.pop('encryption_context', None)
+    file_props.owner = additional_args.pop('owner', None)
+    file_props.group = additional_args.pop('group', None)
+    file_props.permissions = additional_args.pop('permissions', None)
```

Comment:
Our choice to go with `additional_args` in case more things came in is already paying off!